src/include/rbd/librbd.h

```diff
@@ -249,6 +249,10 @@ typedef enum {
   RBD_GROUP_SNAP_STATE_COMPLETE
 } rbd_group_snap_state_t;
 
+typedef enum {
+  RBD_GROUP_SNAP_NAMESPACE_TYPE_USER
```

Comment:
```suggestion
  RBD_GROUP_SNAP_NAMESPACE_TYPE_USER = 0
```

Nit. For clarity and for when the other values will be added later.